doc/users/whats_new/updated_fill_betweenx.rst

```diff
@@ -0,0 +1,7 @@
+Interpolation in fill_betweenx
+------------------------------
+
+The ``interpolate`` parameter now exists for the method :func:`fill_betweenx`.  This allows
+a user to interpolate the data and fill the areas in the crossover points, similarly to :func:`fill_between`.
+::
```

Comment:
The `::` indicates there will be a code block following; it should be removed (or an example added.)